azext_iot/tests/dps/test_iot_dps_int.py

```diff
@@ -46,7 +46,8 @@ def test_dps_enrollment_tpm_lifecycle(self):
                     "iot dps enrollment create --enrollment-id {} --attestation-type {}"
```

Comment:
Not in this iteration. We should breakout/separate DPS unit and integration tests similar to the work we have done for IoT Hub.